osdep/mac/application.h

```diff
@@ -15,10 +15,10 @@
  * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#ifndef MPV_MACOSX_APPLICATION
-#define MPV_MACOSX_APPLICATION
+#ifndef MAC_APPLICATION
+#define MAC_APPLICATION
```

Comment:
not sure about this. we are generally very inconsistent here. we have different prefixes, MPV_*, MP_*, MPLAYER_*. some are suffixed with *_H or *_H_. some don't have either.

Reply:
I wouldn't worry about this much. Also I think nowadays `#pragma once` is the way to go.